aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js

```diff
@@ -21,25 +21,24 @@ const demoStyle = {
 
 export const PageContainerInteractive = () => (
   <AppContainer
-    /* `as`, `title`, and `demoStyle` for Design System site demonstration.
-         Remove from your implementation */
+    // `as`, `title`, and `demoStyle` for Design System site demonstration.
+    //  Remove from your implementation.
     as={ContentArea}
     title="AppContainer"
     {...demoStyle}
   >
     {/* ContentArea is specific for the Design System site and is used
-     * as a visual aid to help present layout concepts. Remove from your
-     * implementation.
-     */}
+     as a visual aid to help present layout concepts. Remove from your
```

Comment:
should we add `Content Area` just to not get confused

Reply:
Done.